sdk/core/Azure.Core/tests/HttpClientTransportFunctionalTest.cs

```diff
@@ -18,23 +26,31 @@ static HttpClientTransportFunctionalTest()
 #if NET461
             ServicePointManager.ServerCertificateValidationCallback += (_, _, _, _) => true;
 #endif
+
```

Comment:
nit: rem but consider to comment above `// No way to disable SSL check per HttpClient on NET461`. Does it hurt this feature?

Reply:
It does - The options are to set the callback per-request (which is what the current behavior is, or to set the callback on ServicePoint like we do now, but using the options delegate.

Reply:
Hopefully, we are good in the default case. On net4x we use the HttpWebReqTransport and if we don't allow direct HttpClientTransport creation the only customers affected are the ones that disabled the  HttpWebReqTransport

Reply:
Discussed offline - this edge case can be solved with customer guidance to set `ServicePointManager.ServerCertificateValidationCallback`